client/src/components/user-boards/board-table/BoardTable.tsx

```diff
@@ -42,6 +46,13 @@ const BoardTable: React.FC<IBoardTable> = ({ activeTab }) => {
     }
   }, [data]);
 
+  const handleRowClick = (boardId: string) => {
+    router.push(`/user-boards/${boardId}/slides/0`);
```

Comment:
Change to 
```suggestion
    router.push(`/user-boards/${boardId}/slides/1`);
```
because after last refactor backend and frontend use 1-based system (i.e. slides have numbers 1, 2, 3, ...)

Reply:
Okay, done